src/transformers/generation/logits_process.py

```diff
@@ -276,9 +276,15 @@ class RepetitionPenaltyLogitsProcessor(LogitsProcessor):
     selected. The formula can be seen in the original [paper](https://arxiv.org/pdf/1909.05858.pdf). According to the
     paper a penalty of around 1.2 yields a good balance between truthful generation and lack of repetition.
 
+    While the intended usage within the paper is to penalize and thus reduce repetition in the generated sequence, this
+    technique can also be used to reward and thus encourage repetition in a similar manner. To penalize and reduce 
```

Comment:
```suggestion
    This technique can also be used to reward and thus encourage repetition in a similar manner. To penalize and reduce 
```
1. Let's trim the intended usage part, most users are only concerned about practical uses :) Short and clear docstring = better!
2. You may need to run `make fixup` after applying the suggested change :)